solidity/test/bridge/VendingMachine.test.ts

```diff
@@ -51,15 +54,15 @@ describe("VendingMachine", () => {
       .mint(await tokenHolder.getAddress(), initialBalance)
 
     await vendingMachine
-      .connect(deployer)
+      .connect(keepCommunityMultiSig)
       .transferOwnership(await governance.getAddress())
```

Comment:
Instead of transferring the role, I would rather update the tests to use `keepCommunityMultiSig` where needed. The same way as it is done in `VendingMachine.Upgrade.test.ts`.

Reply:
4aa598faf485d7e1e678924d2efd34e326d1d9a6